src/kernel_config_manager.sh

```diff
@@ -131,7 +131,7 @@ function save_config_file()
   if [[ "$ret" == 1 ]]; then
     warning "Warning: $name: there's nothing new in this file"
   elif [[ "$ret" -gt 0 ]]; then
-    fail "Could not save user config files"
+    complain "Could not save user config files"
```

Comment:
Nice catch! Since we are here, could you replace those double quotes with single quotes?